v21.2/connect-to-the-database.md

```diff
@@ -1,147 +1,899 @@
 ---
-title: Connect to the Database
+title: Connect to a CockroachDB Cluster
 summary: How to connect to a CockroachDB cluster from your application
 toc: true
 ---
 
-<div class="filters filters-big clearfix">
-  <button class="filter-button current">Local</button>
-  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.db }}</button></a>
+This page documents the required connection configuration for [fully-supported third-party tools](third-party-database-tools.html).
+
+For a list of all supported cluster connection parameters, see the [`cockroach` Connection Parameters](connection-parameters.html).
+
+For a list of community-supported third-party tools, see [Third-Party Tools Supported by the Community](community-tooling.html). CockroachDB supports both native drivers and the PostgreSQL wire protocol. Most client drivers and ORM frameworks connect to CockroachDB like they connect to PostgreSQL.
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="serverless">{{ site.data.products.serverless-plan }}</button>
```

Comment:
Any particular reason we're using `{{ site.data.products.serverless-plan }}` and not `{{ site.data.products.serverless }}`?

Reply:
`{{ site.data.products.serverless-plan }}` maps to "CockroachDB Serverless" and `{{ site.data.products.serverless }}` maps to "CockroachDB Serverless (beta)"

While more accurate, "CockroachDB Serverless (beta)" is a little unnecessarily bulky to be throwing around every time we mention serverless (especially in things like tab titles). But that's just my opinion.

@ianjevans Is there a set convention for when to use `{{ site.data.products.serverless-plan }}` vs `{{ site.data.products.serverless}}`?

Reply:
The intent was to separate the name of the product from what type of cluster the user would see in the Cloud UI. But in practice it has turned into a short name for the product.

The problem with that is eventually we're going to drop the "(beta)" from the product name and the two variables are going to be identical.

Now I'm wondering if we really want "(beta)" there at all.

Reply:
@ianjevans I think we want the beta there for now while the product is still in beta, but the nice thing is once it's out of beta, we just remove 7 characters from 1 file and it gets updated across the board.

Like I wrote in Slack, I think we should stick with just `serverless` and repurpose `serverless-plan` for what it was originally intended to be, which I still don't fully understand what that is 😄 

As bulky as it is, we will eventually move out of beta, and if for some reason the product name changes again, at least we'll be able to update it rather effortlessly. Regardless, we're going to have to do a universal find and replace of `serverless-plan` if we've been using them interchangeably, so probably keeping it for now will not make much of a difference in the end.